### PR TITLE
Added the new functionality for Add "Remove contentstore" in ckeditor.

### PR DESCRIPTION
**Pull request by cfmitrah**

### CODE REVIEW

File: modules/contentbox/modules/contentbox-admin/views/contentStore/editorSelectorHelper.cfm [hunk 1]
```diff
@@ -35,8 +35,10 @@ function pagerLink(page){
 	} );
 }
 function insertContentStore(slug){
-	var customContent = "{{{ContentStore slug='"+slug+"'}}}";
-	insertEditorContent( '#rc.editorName#', customContent );
+	var widgetContent = new CKEDITOR.dom.element( 'contentstore' );
```
Comment (lmajano) [inline modules/contentbox/modules/contentbox-admin/views/contentStore/editorSelectorHelper.cfm]: Unfortunately, this bleeds the implementtion of the editor used.  As you can see, this file has no concept of ckeditor or any other editor. Meaning that each editor module provides methods so you can interact with it, so this file reamins editor agnostic.  This bleeds into direct ckeditor.  

Look in the ckeditor module to see the methods we provide

Comment (lmajano) [reply modules/contentbox/modules/contentbox-admin/views/contentStore/editorSelectorHelper.cfm]: Any updates on this @cfmitrah ?

Comment (lmajano) [reply modules/contentbox/modules/contentbox-admin/views/contentStore/editorSelectorHelper.cfm]: Any updates on this @cfmitrah ?

**Comment by lmajano** [comment]
Can I get an update on this @cfmitrah  if not, I have to close

**Comment by cfmitrah** [comment]
@lmajano I will close it for now

**Pull request closed without merging**